our-work/software/provident.yml

```diff
@@ -0,0 +1,15 @@
+title: Provident
+description: A web app to support the PROVIDENT research study looking to prevent drug-related deaths in neighborhoods across Rhode Island. The dashboard showcases a predictive analytics model that identifies neighborhoods where future overdose deaths are most likely to occur. We are also tracking interventions from organizations and what effects they will have on the overdose data. The results from this study will improve public health decision-making and inform resource allocation to communities that should be prioritized for evidence-based prevention, treatment, recovery, and overdose rescue services.
```

Comment:
I have always heard "opioid" related, is drug related too general?

Reply:
I grabbed this line from the provident website's bite. I think that drug-related is fine

`The PROVIDENT Study is a research study looking to prevent drug-related deaths in neighborhoods across Rhode Island.`
[Source](https://provident.preventoverdoseri.org/)